dbt-bigquery/tests/conftest.py

```diff
@@ -31,7 +31,7 @@ def oauth_target():
         "method": "oauth",
         "threads": 1,
         "job_retries": 2,
-        "dataproc_region": os.getenv("DATAPROC_REGION"),
+        "compute_region": os.getenv("COMPUTE_REGION"),
```

Comment:
CI uses the workflow on `main`, which does not pass in this variable (yet). Revert this line and the line below.
```suggestion
        "compute_region": os.getenv("COMPUTE_REGION") or os.getenv("DATAPROC_REGION"),
```